NSS-Likeable-Lemurs/WorkforceManagement
# Ticket#7

## Link to Ticket
Closes # [Link](https://github.com/NSS-Likeable-Lemurs/WorkforceManagement/issues/7)

## Description of Proposed Changes
- Create a view  model for DepartmentDetail
- Created mvc for DepartmentDetail

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <ticket#7>
To Test: Run the Code - click on the Department Link in Navbar then Click on Detail. The Department detail should display the department  name with employees assigned to that department.
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* DepartmentController, Department View, Department View Model, and Department Model
## Mentions @username

@HelenChalmers 
@streatorward 
@kellycook301 
@Akeen11 

## Definition of Done

The project must be fully documented. This includes the following:
Each method is commented with Author, Purpose, and Methods
2 The Department link on the Navbar - when pressed shows the Department's Name, Budget and Number of employees.
3. When user hits Detail Department Link, user should be able to view department  name and employees assigned to that department.
4.Every line of code has been peer reviewed.
5. Integration Testing - The Detail Test for department run and work.
